index.html

```diff
@@ -1144,11 +1107,18 @@ <h2>Dereferencing the Secondary Resource</h2>
 					case when dereferencing it returns an HTTP <code>3xx</code> (Redirection) response with a
 					<code>Location</code> header (see section 7.1.2 of [[RFC7231]].
 					</p>
-					</li>
 				</ol>
-				<li>Otherwise, dereference the secondary resource as defined by the media type ([[RFC2046]]) of the primary resource.
-				</li>
-				</li>
+				<li>Otherwise, dereference the secondary resource identified by the DID fragment as defined by the media type ([[RFC2046]]) of the primary resource.
+				For example, if the primary resource is a representation of a DID document with media type <code>application/did+ld+json</code>, then
+				the fragement is treated according to the rules associated with the
```

Comment:
sp: fragment

Reply:
Making the fix easy to apply...
```suggestion
				the fragment is treated according to the rules associated with the
```

Reply:
Ah, thanks! I'm not yet a heavy user of github's UI, so I'm still learning!